sites/docs/src/content/installation/manual.md

```diff
@@ -16,7 +16,7 @@ description: How to setup shadcn-svelte manually.
 Use the `svelte-add` CLI to add Tailwind CSS to your project.
 
 ```bash
-npx @svelte-add/tailwindcss@latest
+npx svelte-add@latest tailwindcss
```

Comment:
I've been using it without `@latest` and it's been working just fine since we started pinning the dependency versions

Reply:
yea I think it's probably fine with `npm` and `pnpm`, but I still worry about the aggressive caching strategy of the other package managers, like `bun` who's biggest offender of this

even trying it now, it's giving me `2.3.3` of `svelte-add` 🙃
![img](https://i.imgur.com/T5XzzTC.png)